openslides/motions/static/templates/motions/motion-form.html

```diff
@@ -55,34 +44,45 @@ <h1 ng-if="!motion.id" translate>New motion</h1>
   </div>
   <div class="form-group">
     <label for="selectTags" translate>Tags</label>
-    <select multiple ng-options="tag.id as tag.name for tag in tags"
-        ng-model="motion.tags_id" class="form-control" name="selectTags">
-    </select>
+    <ui-select multiple ng-model="motion.tags_id">
+      <ui-select-match placeholder="{{ 'Select or search a tag...' | translate }}">
+        {{ $item.name }}
+      </ui-select-match>
+      <ui-select-choices repeat="tag.id as tag in tags | filter: $select.search">
+        {{ tag.name }}
+      </ui-select-choices>
+    </ui-select>
   </div>
   <div class="form-group">
     <label for="selectAttachments" translate>Attachments</label>
-    <select ng-options="file.id as file.title for file in mediafiles"
-        ng-model="motion.attachments_id" class="form-control" name="selectAttachments">
-    </select>
+    <ui-select multiple ng-model="motion.attachments_id">
+      <ui-select-match placeholder="{{ 'Select or search an attachment...' | translate }}">
+        {{ $item.name }}
+      </ui-select-match>
+      <ui-select-choices repeat="file.id as file in mediafiles | filter: $select.search">
+        {{ file.title }}
+      </ui-select-choices>
+    </ui-select>
   </div>
-  <!-- TODO: show only if supporters is enabled -->
-  <div class="form-group">
+  <!-- show supporters if enabled -->
+  <div class="form-group" ng-if="config('motions_min_supporters') > 0">
     <label for="selectSupporter" translate>Supporters</label>
-    <ui-select multiple ng-model="motion.supporters_id" theme="bootstrap" name="selectSupporter">
-      <ui-select-match placeholder="{{ 'Select or search a participant...' | translate }}">
-        {{ $select.selected.get_short_name() }}
+    <ui-select multiple ng-model="motion.supporters_id">
+      <ui-select-match placeholder="{{ 'Select or search a supporter...' | translate }}">
+        {{ $item.get_full_name() }}
       </ui-select-match>
       <ui-select-choices repeat="user.id as user in users | filter: $select.search">
-        <div ng-bind-html="user.get_short_name() | highlight: $select.search"></div>
-        <small ng-bind-html="user.structure_level | highlight: $select.search"></small>
+        <div ng-bind-html="user.get_full_name() | highlight: $select.search"></div>
       </ui-select-choices>
     </ui-select>
   </div>
-  <!-- TODO: preselect default workflow -->
+  <!-- TODO: preselect default workflow
+  does not work: ng-init="motion.workflow = config('motions_workflow')"-->
```

Comment:
I would fix this in the MotionCreateCtrl

$scope.motion = {workflow: config('motions_workflow')}